middleware/kubernetes/README.md

```diff
@@ -121,6 +121,59 @@ kubernetes coredns.local {
 	# Each line consists of the name of the federation, and the domain.
 	federation myfed foo.example.com
 	
+	# autopath [ndots:NDOTS] [resolv:RESOLV-CONF-FILE] [onnxdomain:RESPONSE] ...
```

Comment:
It would have to be....

[NDOTS [RESOLVE-CONF [RESPONSE]]]

Since they are optional... if a user wants to specify a response, they need to specify the other two options at their default values.